src/RequestExecutor/HttpRequestExecutor.ts

```diff
@@ -186,10 +186,16 @@ export class HttpRequestExecutor implements RequestExecutor {
         break;
       }
     }
+    if (truncated) {
```

Comment:
Please put a blank line here
```suggestion
    
    if (truncated) {
```